tests/test_prescan_sub.py

```diff
@@ -321,10 +318,11 @@ def test_prescan_sub():
             corgidrp_result = output_dataset[0].data
             iit_result = iit_frames[0] if return_full_frame else iit_images[0]
 
-            if not corgidrp_result == approx(iit_result):
+            if not np.nanmax(corgidrp_result-iit_result) < tol:
```

Comment:
Change to np.abs() > tol. It works without this b/c the quantity turns out to be positive, but this change would make it clearer to someone seeing this test fresh that the subtraction is not negative, thus avoiding the tolerance check.